### DISCUSSION TITLE
quiz error

### DISCUSSION THREAD

**Discussion by sendhelpp** [sst-inc/help#36] [Q&A - iOS] [answered]
Thread 1: "[<UIView 0x123705d30> setValue:forUndefinedKey:]: this class is not key value coding-compliant for the key scoreLabel2." 
error keeps appearing and idk why
idk why
changed the program around in an attempt to fix it so it may be a bit broken.
[the nikola quiz.virus.zip](https://github.com/sst-inc/help/files/7068017/the.nikola.quiz.virus.zip)

**Comment by granwyntan** [comment, marked as answer]
<img width="181" alt="Screenshot 2021-09-01 at 12 03 53" src="https://user-images.githubusercontent.com/61010627/131609972-9c500e32-b5cb-4574-9bea-27257b8787b9.png">
Hi @sendhelpp, you have 2 outlets of the same name, so you could remove 1 to fix the issue
Here's the compressed project:
<br>
[the nikola quiz.virus.zip](https://github.com/sst-inc/help/files/7088155/the.nikola.quiz.virus.zip)

Hope it helps!

**Reply by Joeispro99** [reply]
[the nikola quiz.virus.zip](https://github.com/sst-inc/help/files/7088160/the.nikola.quiz.virus.zip)

**Reply by sendhelpp** [reply]
it worked but now it crashes because of another problem which is: Thread 1: "-[UIView buttonTapped:]: unrecognized selector sent to instance 0x152521610" when it tries to carry out the unwind segue

**Reply by Joeispro99** [reply]
When you click your play again button, it is connected to a Results View Controller, I reconnected the IBAction to the first ResultsViewController (not too sure why there are 2 options when creating an IBAction @granwyntan) and then the play again button works again.

**Reply by Joeispro99** [reply]
Just a follow up but has your app worked already? @sendhelpp

**Reply by sendhelpp** [reply]
uh yes I think
